docs/getting_started.md

```diff
@@ -52,13 +52,13 @@ cargo run
 Note: You may also pass environment variable values in the same command as the run command. This is especially useful for setting log levels.
 
 ```sh
-RUST_LOG=debug cargo run -p trin
+RUST_LOG=debug cargo run 
```

Comment:
Any idea the history here of why it used to be `-p trin`?

Reply:
Not sure, but the two commands are identical, so I prefer the more succinct one. I wish I could remember if I even had a reason to change that in the first place 🤔.